.cbmc-batch/jobs/aws_array_list_set_at/cbmc-batch.yaml

```diff
@@ -1,4 +1,4 @@
 jobos: ubuntu16
-cbmcflags: "--bounds-check;--pointer-check;--function;aws_array_list_set_at_harness;--unwindset;strlen.0:33,memcpy.0:33,__builtin___memcpy_chk.0:33,aws_mem_release:1;--unwinding-assertions"
+cbmcflags: "--bounds-check;--pointer-check;--function;aws_array_list_set_at_harness;--unwindset;strlen.0:5,memcpy.0:5,__builtin___memcpy_chk.0:5,aws_mem_release:1;--unwinding-assertions"
```

Comment:
Action item: not for this CR, but should go on our roadmap. We should use the makefile to calculate the loop unrolling bounds from the data-structure sizes, rather than hand calculating them.  It would make the proofs more maintainable in the future